.internal-ci/helm/fog-ledger/values.yaml

```diff
@@ -0,0 +1,234 @@
+      test:
+        shardSize: 400_000
+        exceedBlockHeightBy: 10_000
+        shardOverlap: 0
+        count: 2
+        blockHeightRetrieval:
+          blockCountURL: https://node1.test.mobilecoin.com/gw/consensus_common.BlockchainAPI/GetLastBlockInfo
+          responseJQ: '.index'
+          queryInterval: 5m
+          requestBody: ''
+      main:
+        shardSize: 400_000
+        exceedBlockHeightBy: 10_000
+        shardOverlap: 0
+        count: 3
+        blockHeightRetrieval:
+          blockCountURL: https://node1.prod.mobilecoinww.com/gw/consensus_common.BlockchainAPI/GetLastBlockInfo
+          responseJQ: '.index'
+          queryInterval: 5m
+          requestBody: ''
+
+  router:
+    ### list of fog-ledger-router hostnames (client responder ID)
+    hosts:
+    - partner: ''
+      responderID: ''
+
+    replicaCount: 1
+
+    image:
+      org: ''
+      name: fog-ledger
+      pullPolicy: Always
+
+    resources:
+      limits:
+        intel.com/sgx: 5000
+        memory: 5Gi
+      requests:
+        intel.com/sgx: 5000
+        memory: 5Gi
+        cpu: 1100m
+
+    nodeSelector:
+      sgx-enabled-node: 'true'
+
+    tolerations:
+    - key: sgx
+      operator: Equal
+      value: 'true'
+      effect: NoSchedule
+
+    # disable affinity rules for single node testing
+    podManagementPolicy: Parallel
+    affinityEnabled: true
+    topologySpreadConstraintsEnabled: true
+
+    rust:
+      backtrace: full
+      log: info,rustls=warn,hyper=warn,tokio_reactor=warn,mio=warn,want=warn,reqwest=warn,rusoto_core=error,rusoto_signature=error,h2=error,rocket=warn,<unknown>=warn
+
+    persistence:
+      enabled: true
+      spec:
+        storageClassName: fast
+        accessModes:
+        - 'ReadWriteOnce'
+        resources:
+          requests:
+            storage: 512Gi
+
+    podAnnotations:
+      fluentbit.io/include: 'true' # collect logs with fluentbit
+      fluentbit.io/exclude-jaeger-agent: 'true'
+
+    ingress:
+      enabled: true
+      common:
+        tls:
+          clusterIssuer: letsencrypt-production-http
+        blocklist:
+          enabled: true
+          pattern: patterns/blocked-countries
+        annotations: |-
+          haproxy.org/server-ssl: "false"             # The backend (server) is http
+          haproxy.org/timeout-client: 239s            # 4 min timeout on azure
+          haproxy.org/timeout-server: 239s
+          haproxy.org/timeout-http-keep-alive: 120s
+          haproxy.org/abortonclose: "true"
+          haproxy.org/backend-config-snippet: |-
+            http-reuse aggressive
+            dynamic-cookie-key {{ include "fog-ledger.grpcCookieSalt" . }}
+            cookie "LEDGER" insert indirect nocache dynamic
+
+      grpc:
+        annotations: |-
+          haproxy.org/server-proto: "h2"              # Force GRPC/H2 mode
+
+      http:
+        annotations: |-
+          haproxy.org/path-rewrite: '/gw/(.*) /\1'    # Strip the /gw prefix
```

Comment:
what does `gw` stand for, noticed that it was there on a few of the urls

Reply:
`gateway`, basically the path for the go-grpc-gateway, grpc/json gateway instance.